app/renderer/app_status.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const EventEmitter = require('events');
+const EventEmitter = require("events");
```

Comment:
Unnecessary change

Reply:
this was unprecedented. its probably the prettier in my editor